scripts/generate.js

```diff
@@ -4,8 +4,8 @@
 const { copyDocs, copyDist } = require('./_util/copy');
 const { generateAll } = require('./_generate');
 
-(function () {
-    copyDist();
-    copyDocs();
-    generateAll();
+(async function () {
+    await copyDist();
+    await copyDocs();
+    await generateAll();
```

Comment:
HTML generation and file download `copy` behavior conflict if they run at the same time. Doing this ensures that `copyDocs()` and `generateAll()` don't run at the same time.